docs/installation/users.rst

```diff
@@ -5,82 +5,83 @@ For the users
 
 .. note::
 
-   If you encounter an issue during installation, see our
-   `troubleshooting section <instillation_troubleshooting>`_.
+  If you encounter an issue during installation, see our
+  `troubleshooting section <instillation_troubleshooting>`_.
+  If that doesn't resolve your issue, please create a bug report on our
+  `Github issue tracker <https://github.com/deepanshs/mrsimulator/issues>`_.\
 
-   If that doesn't resolve your issue, please create a bug report on our
-   `Github issue tracker <https://github.com/deepanshs/mrsimulator/issues>`_.
 
 Strict Requirements
 -------------------
 
 ``mrsimulator`` has the following strict requirements:
 
-- `Python <https://www.python.org>`_ 3.6 or later
+- `Python <https://www.python.org>`_ 3.7 or later
 - `Numpy <https://numpy.org>`_ 1.17 or later
 
 See :ref:`requirements` for a full list of requirements.
 
-Make sure you have the required version of python by typing the following in the terminal,
+Make sure you have the required version of Python by typing the following in the terminal,
 
 .. code-block:: shell
 
       $ python --version
 
-For *Mac* users, python version 3 is installed under the name *python3*. You may replace
-*python* for *python3* in the above command and all subsequent python statements.
+For *MacOS* users, Python version 3 is installed under the name *python3*. You may
+replace *python* for *python3* in the above command and all subsequent Python statements.
 
 For *Windows* users, Python is not usually installed by default. See
-`Python.org <https://www.python.org/downloads/windows/>`_ for a list of official Python downloads
-and Windows installation instructions.
+`Python.org <https://www.python.org/downloads/windows/>`_ for a list of official Python
+downloads and Windows installation instructions.
 
 .. seealso::
 
-  If you do not have python or have an older version of python, you may visit the
+  If you do not have Python or have an older version of Python, you may visit the
   `Python downloads <https://www.python.org/downloads/>`_ or
   `Anaconda <https://www.anaconda.com/products/individual/>`_ websites and follow their
```

Comment:
Should we really mention Anaconda here? @mccarthy677 Do you remember any issues with python.org vs Anaconda Python on windows?

Reply:
Will resolve any issues relating to instillation in a future PR.